wallet/src/main/scala/org/bitcoins/wallet/models/DLCTable.scala

```diff
@@ -0,0 +1,173 @@
+package org.bitcoins.wallet.models
```

Comment:
This should be reviewed, I think I am storing all the necessary data but could be missing something